### PR TITLE
Fix query for find and match.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

**Review by gannan08 — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # bedrock-web-vc-store ChangeLog
 
+## 4.0.0 - 2021-05-13
```
Comment (mattcollier) [inline CHANGELOG.md:3]:
```suggestion
## 4.0.0 - TBD
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+### Changed
+- **BREAKING**: Updates `find` params to now only accept `query`.
+- Updates `find` and `match` to now handle an array of `types`.
```
Comment (mattcollier) [inline CHANGELOG.md:7]:
```suggestion
- **BREAKING**: Updates `find` API params to only accept `query`.
- Updates `find` and `match` APIs to handle an array of `types`.
```

File: lib/VerifiableCredentialStore.js [hunk 3]
```diff
@@ -40,28 +40,27 @@ export default class VerifiableCredentialStore {
...
    */
-  async find({type, issuer, displayable}) {
-    const equals = {};
-    if(type) {
-      if(Array.isArray(type)) {
-        // needs to be implemented on the edv-client side first
-        throw new Error('"type" as array is not implemented.');
-      } else {
-        equals['content.type'] = type;
+  async find({query} = {}) {
```
Comment (mattcollier) [inline lib/VerifiableCredentialStore.js:47]: this is setup to allow `query` to be `undefined`, but this code breaks on L50 if query === `undefined`.  It looks to me that query is a required parameter.

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: lib/VerifiableCredentialStore.js (hunk 3 above)
Comment (dlongley) [reply lib/VerifiableCredentialStore.js:47]: Yeah, we could do `{query = []} = {}` to set it by default, but better to throw if it isn't there. Good catch.

**Review by dlongley — APPROVED** [review]
Approving as long as @mattcollier's comments are addressed. Thanks!

File: lib/VerifiableCredentialStore.js [hunk 4]
```diff
@@ -40,28 +40,30 @@ export default class VerifiableCredentialStore {
...
-        throw new Error('"type" as array is not implemented.');
-      } else {
-        equals['content.type'] = type;
-      }
-    }
-    if(issuer) {
-      equals['meta.issuer'] = issuer;
+  async find({query} = {}) {
+    if(!query) {
+      throw new Error('"query" MUST be defined.');
```
Comment (mattcollier) [inline lib/VerifiableCredentialStore.js:49]: No need to yell.  :)
```suggestion
      throw new TypeError('"query" is a required parameter.');
```

**Review by mattcollier — APPROVED** [review]
TY, one more suggestion about verbiage.

**Pull request merged**